cogs/reminder.py

```diff
@@ -1,38 +1,145 @@
+import asyncio
+import logging
+from typing import Mapping
+
+from tortoise import timezone
 import discord
+from discord import app_commands
 from discord.ext import commands
+from db.models import MemberReminder
+
+from utils.commands import available_subcommands
+from utils.converters import DatetimeConverter
+from utils.misc import DEGENERACY_DELAY, long_sleep_until, \
+    discord_timestamp_string_format as dts_fmt
+from utils.pagination import paginated_embed_menus, PaginationView
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)  # TODO: Change back to logging.INFO
```

Comment:
Discussion about the need to set the level of debugging for each file
Can be addressed by setting a `cogs` logger in `bot.py`